### PR TITLE
Variable names are standardized for easier maintenance in the future.

### PR DESCRIPTION
**Pull request by jvanasco**
The current code uses `$data` and `$default_sec` for both "rc" and "ini" files, sometimes redeclaring a variable from one context to the other in the same scope/closure.  Also, two types of prefixes are used to contextually identify the variables: `rc` (no underscore) and `ini_` (with underscore).

The following changes are proposed in this PR:  (No functionality is added or altered)

* `$rcfile` -> `$rc_file` (same style as `$ini_file`)
* `$data` -> `$rc_data` or `$ini_data`
* `$default_sec` -> `$rc_default_sec` or `$ini_default_sec`

These changes should make it much easier for contributors to review code.

### CODE REVIEW

File: bin/opm [hunk 1]
```diff
@@ -2003,13 +2003,13 @@ sub init_installation_ctx ($) {
 sub upgrade_target ($$$$) {
     my ($ctx, $account, $name, $meta_file) = @_;
 
-    my $data = read_ini $meta_file;
-    my $default_sec = $data->{default};
+    my $ini_data = read_ini $meta_file;
+    my $rc_default_sec = $ini_data->{default};
```
Comment (agentzh) [inline bin/opm]: This should be `ini_default_sec`, right?

**Comment by agentzh** [comment]
@doujiang24 @membphis Will you have a look at this PR as well? Thanks!

File: bin/opm (hunk 1 above)
Comment (jvanasco) [inline bin/opm]: Yes. I don't know how that slipped through!

**Comment by doujiang24** [comment]
@agentzh LGTM :)

**Comment by agentzh** [comment]
@jvanasco Just merged. Sorry for the delay on my side and thanks for the patch!

**Comment by jvanasco** [comment]
thanks!  i'll rebase the `build-install` feature against this!

**Pull request closed without merging**